ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Server/appsettings.json

```diff
@@ -38,16 +38,16 @@
 //  },
 ////#endif
 ////#if (GenerateApiOrGraph)
-//  "CalledApi": {
+//  "DownstreamAPI": {
 //    /*
-//     'CalledApiScopes' contains space separated scopes of the Web API you want to call. This can be:
+//     'Scopes' contains space separated scopes of the Web API you want to call. This can be:
 //      - a scope for a V2 application (for instance api://b3682cc7-8b30-4bd2-aaba-080c6bf0fd31/access_as_user)
 //      - a scope corresponding to a V1 application (for instance <App ID URI>/.default, where  <App ID URI> is the
 //        App ID URI of a legacy v1 Web application
 //      Applications are registered in the https://portal.azure.com portal.
 //    */
-//    "CalledApiScopes": "user.read",
-//    "CalledApiUrl": "[WebApiUrl]"
+//    "BaseUrl": "[WebApiUrl]",
+//    "Scopes": "user.read"
```

Comment:
nit: in the blazor server one, this is called "WebApiUrl" not "BaseUrl" should they be consistent?

Reply:
this would also rename the tag (it's based on renaming in a file)